### PR TITLE
Copy Group Feature

### PR DESCRIPTION
**Pull request by PCOffline**

### CODE REVIEW

File: src/components/shared/users-input-fields/UsersInputFields.jsx [hunk 1]
```diff
@@ -25,14 +25,15 @@ const UsersInputFields = ({
...
 }) => {
   const classes = useStyles();
   const { t } = useTranslation();
   const { enqueueSnackbar } = useSnackbar();
   const currentUser = useContext(userContext);
-  const [selectedUser, setSelectedUser] = useState(undefined);
+  const [selectedOption, setSelectedOption] = useState(null);
   const [searchedUsers, setSearchedUsers] = useState(groupUsers);
-  const [isModeSearch, setIsModeSearch] = useState(true);
+  const [isModeSearch, setIsModeSearch] = useState('search');
```
Comment (carmeltsadok) [inline src/components/shared/users-input-fields/UsersInputFields.jsx:36]: Why?

File: src/components/shared/users-autocomplete/UsersAutocomplete.styles.js [hunk 2]
```diff
@@ -38,11 +38,63 @@ const useStyles = makeStyles((theme) => ({
...
+    boxShadow: 'none',
+    direction: 'rtl',
+    display: 'flex',
+    wordBreak: 'break-all',
+    '&:hover': {
+      background: theme.palette.hover.main,
+      boxShadow: 'none',
+    },
+    justifyContent: 'space-between',
+    flexDirection: 'row',
```
Comment (carmeltsadok) [inline src/components/shared/users-autocomplete/UsersAutocomplete.styles.js:57]:
```suggestion
```

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: src/components/edit-group/edit-users-list-page/EditUsersListPage.jsx [hunk 3]
```diff
@@ -15,12 +15,10 @@ const EditUsersListPage = ({ group, setGroup }) => {
   const [removeUserLoaders, setRemoveUserLoaders] = useState([]);
   const [updateUserLoaders, setUpdateUserLoaders] = useState([]);
 
-  const handleAddUser = (userToAdd, role) => {
+  const handleAddUser = (userToAdd, role, supressSnackbar) => {
```
[the remark this answers is not in the document]
Comment (PCOffline) [reply src/components/edit-group/edit-users-list-page/EditUsersListPage.jsx]: Fixed

File: src/components/shared/users-autocomplete/UsersAutocomplete.jsx [hunk 4]
```diff
@@ -1,47 +1,80 @@
...
-import { Card, Typography } from '@material-ui/core';
+import Card from '@material-ui/core/Card';
+import Typography from '@material-ui/core/Typography';
+import Group from '@material-ui/icons/Group';
+import Person from '@material-ui/icons/Person';
+import { useTranslation } from 'react-i18next';
 import useStyles from './UsersAutocomplete.styles';
+import LockIcon from '../lock-icon/LockIcon';
+
+const tagLength = 4;
```
[the remark this answers is not in the document]
Comment (PCOffline) [reply src/components/shared/users-autocomplete/UsersAutocomplete.jsx]: Fixed

File: src/components/shared/users-input-fields/UsersInputFields.jsx [hunk 5]
```diff
@@ -69,63 +93,75 @@ const UsersInputFields = ({
...
 
   const SearchBar = isModeSearch ? SearchUserBar : AddUserSearchBar;
 
   return (
     <div className={classes.root}>
       <div className={classes.inputArea}>
-        <ButtonGroup className={classes.buttonGroup} variant="primary" color="primary">
+        <ButtonGroup
+          className={classes.buttonGroup}
+          variant="primary"
```
[the remark this answers is not in the document]
Comment (PCOffline) [reply src/components/shared/users-input-fields/UsersInputFields.jsx:126]: We already talked about that, and we agreed to keep it. Without the invalid variant, it looks very ugly

File: src/i18n.js [hunk 6]
```diff
@@ -54,6 +54,8 @@ const resources = {
         file: 'יש לבחור תמונה',
         server: ':) קרתה תקלה.. נסו שוב מאוחר יותר',
         userAlreadyExist: 'החבר/ה כבר בקבוצה',
+        someUsersAlreadyExist: 'חלק מהחברים כבר בקבוצה',
+        allUsersAlreadyExist: 'כל החברים כבר בקבוצה',
```
[the remark this answers is not in the document]
Comment (PCOffline) [reply src/i18n.js]: Removed the someUsersAlreadyExist. We need the allusersAlreadyExist though

File: src/components/shared/users-input-fields/UsersInputFields.jsx (hunk 1 above)
Comment (PCOffline) [reply src/components/shared/users-input-fields/UsersInputFields.jsx]: Fixed

**Review by carmeltsadok — CHANGES_REQUESTED** [review]
- Fix icons css
- Redirect docs in new tab
- Change members count in add members dialog's message

**Comment by PCOffline** [comment]
@carmeltsadok Should I remove the `suppressSnackbar` and instead just instantly call the `onAdd` function from the `onGroupSelect`? Or is it safer to just call the `onUserSelect` method, if we ever add more functions and checks in it?

File: src/services/DocsService.js [hunk 7]
```diff
@@ -4,6 +4,10 @@ class DocsService {
   static async redirect() {
     window.location.replace(`${config.uri.api_gateway_uri}docs`);
   }
```
Comment (carmeltsadok) [inline src/services/DocsService.js:6]: remove

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: src/services/DocsService.js (hunk 7 above)
Comment (PCOffline) [reply src/services/DocsService.js:6]: done

**Pull request merged**